examples/USER/phonon/dynamical_matrix_command/README.md

```diff
@@ -0,0 +1,21 @@
+# LAMMPS LATTICE DYNAMICS COMMANDS
+
+## DYNAMICAL MATRIX CALCULATOR
+
+This directory contains the ingredients to calculate a dynamical matrix.  
+
+Example:
+```
+NP=4 #number of processors
+mpirun -np $NP lmp_mpi < in.silicon > out.silicon
```

Comment:
please use `-in in.silicon` and `-log log.silicon` instead of i/o redirection in any examples. i/o redirection does not work correctly with all MPI implementation and is mostly a legacy feature to make the C++ LAMMPS version behave similar to its Fortran predecessors. Our goal is to slowly phase this out in order to simplify the setup of the LAMMPS object and embedding of it in script languages.

also, it would be great to have a second example with multiple atom types (and masses), so it could function as a more specific regression test input.